src/lib_ccx/ccx_encoders_spupng.c

```diff
@@ -650,14 +701,75 @@ int spupng_export_string2png(struct spupng_t *sp, uint32_t *str, FILE* output)
 	
 	FT_GlyphSlot slot = face->glyph;
 
+	// For parsing tags
+	int bracket = 0;
+	int end = 0;
+	int bold = 0;
+	int italics = 0;
+	FT_Face current_face = face;
+
 	// Render characters to image
 	for (uint32_t *iter = str; *iter; ++iter)
 	{
 		uint32_t current_char_code = BigtoLittle32(*iter); // Convert big-endian and little-endian
 
-		if (FT_Load_Char(face, current_char_code, FT_LOAD_RENDER)) continue; // ignore errors
+		// Parse tags like <i> and <b>
+		if (end) {
+			if (current_char_code == '>') {
```

Comment:
It seems to me that if we meet tags like `<b><i>test</i> test2</b>` (if that is valid), it would output "**_test_** test2" instead of "**_test_** **test2**"